src/components/CategoryCardsLayout/CategoryCardsLayout.jsx

```diff
@@ -1,80 +1,97 @@
+
+import appliances from '../../assets/images/appliances.png';
+import all from '../../assets/images/all.png';
+import { useTranslation } from 'react-i18next';
+
 
 const CategoryCardsLayout = () => {
   const { t } = useTranslation();
 
   return (
-    <Flex
-      justifyContent="center"
-      maxWidth="4xl"
-      mx="auto"
-      mt={['0', '0', '24', '0']}
-    >
-      <Grid
-        maxH="72"
-        maxW="4xl"
-        templateRows="repeat(12, 1fr)"
-        templateColumns="repeat(25, 1fr)"
-        gap={4}
-      >
-        <GridItem borderRadius="xl" colSpan={8} rowSpan={12} bg="#EAEAF1">
-          <CategoryCard
-            categoryPic={sofa}
-            direction="column"
-            categoryName={t('categoryCards.furniture')}
-            ml="4"
-          ></CategoryCard>
-        </GridItem>
-        <GridItem borderRadius="xl" colSpan={5} rowSpan={6} bg="#F6E8CD">
-          <CategoryCard
-            categoryPic={Clothes}
-            direction="column"
-            categoryName={t('categoryCards.clothes')}
-          ></CategoryCard>
-        </GridItem>
-        <GridItem borderRadius="xl" colSpan={5} rowSpan={6} bg="#CBECE9">
-          <CategoryCard
-            categoryPic={book}
-            direction="column"
-            categoryName={t('categoryCards.books')}
-          ></CategoryCard>
-        </GridItem>
-        <GridItem borderRadius="xl" colSpan={7} rowSpan={6} bg="#E6D0EF">
-          <CategoryCard
-            categoryPic={toys}
-            direction="row-reverse"
-            categoryName={t('categoryCards.toys')}
-            mt="24"
-          ></CategoryCard>
-        </GridItem>
-        <GridItem borderRadius="xl" colSpan={7} rowSpan={6} bg="#F0D0D2">
-          <CategoryCard
-            categoryPic={medicalkit}
-            direction="row-reverse"
-            categoryName={t('categoryCards.medics')}
-            mt="24"
-          ></CategoryCard>
-        </GridItem>
-        <GridItem borderRadius="xl" colSpan={10} rowSpan={6} bg="#CFD6F2">
-          <CategoryCard
-            categoryPic={appliances}
-            direction="row-reverse"
-            categoryName={t('categoryCards.appliances')}
-            mt="24"
-          ></CategoryCard>
-        </GridItem>
-      </Grid>
+    <Flex maxW="4xl" m="auto" align="center" justify="space-between">
+      <Stack>
+      <Text fontSize="xl" fontWeight="medium">Categories</Text>
+
+    <Center direction="row" flexWrap="wrap" justifyContent="space-between">
+    <Wrap spacing={["","","14","16"]} align="center">
+  <WrapItem>
+        <CategoryCard
+          categoryPic={furniture}
+          categoryName={t('categoryCards.furniture')}
+          onClick={() => setQuery('furniture')}
+          bgColor="#EAEAF1"
+        ></CategoryCard></WrapItem>
+ <WrapItem>
+        <CategoryCard
+          categoryPic={Clothes}
+          categoryName={t('categoryCards.clothes')}
+          onClick={() => setQuery('clothes')}
```

Comment:
where's `setQuery` coming from? It's causing errors.

Reply:
it is fixed